script/test

```diff
@@ -24,5 +24,6 @@ export DNSIMPLE_TOKEN=
 export DYN_CUSTOMER=
 export DYN_PASSWORD=
 export DYN_USERNAME=
+export GOOGLE_APPLICATION_CREDENTIALS=
```

Comment:
You'll want to do the same in `script/cibuild`

Reply:
OK added to coverage file cause that's the file which  I think you ment.